src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheck.java

```diff
@@ -86,6 +87,7 @@
     public int[] getDefaultTokens() {
         return new int[] {
             TokenTypes.ARRAY_INIT,
+            TokenTypes.AT,
```

Comment:
@romani Please confirm it is ok to be default.
I don't see anything wrong as most people don't write with space after.